File: task-client/src/app/models/task.ts
export class Task {
    taskId?:number;
    title?:string;
    completed?:boolean;
  
    

    constructor(title?:string, completed?:boolean, taskId?:number) {
        this.taskId = taskId
        this.title = title;
        this.completed = completed;
    }
}

File: task-api-2/Models/Task.cs
using System.ComponentModel.DataAnnotations;


namespace task_api_2.Models
{
    public class Task
    {

        public int TaskId { get; set; }
        // task id (ID)
        [Required]
        public string Title { get; set; }
        //title (string)
        [Required]
        public bool Completed { get; set; }
        //completed (boolean)
    }
}